lib/brains/client.py

```diff
@@ -37,10 +37,16 @@ class HttpMock(Http):
 
     def http_do(self, http_verb, full_path, headers, body):
         try:
-            return mocks['{0} {1}'.format(http_verb, full_path)]
+            next_mock = mocks.pop()
+            if next_mock[:2] != (http_verb, full_path):
+                # The mock was called in an order you didn't expect.
+                raise MockError(
+                    'Mock called in wrong order, expecting: {0} {1}'
+                    .format(http_verb, full_path))
+            return next_mock[2:]
```

Comment:
Well i don't have to do it every test, it just happens all the time now, so for the moment I'm happy.